PIC_database_updater.py

```diff
@@ -52,49 +59,71 @@ def update_accession(self, barcode, accession):
                 barcode: barcode of the record to update
                 accession: the accessions number to update the record with"""
 
```

Comment:
this logic is repeated in a lot of the functions, performs a database check if the field already in the database, and only allows update if empty, or if the use decides to manually force update after acknowledging the risks.